po/R-es.po

```diff
@@ -168,27 +168,27 @@ msgstr "Tutoriales"
 #: development.R:20
 #, fuzzy
 msgid "Released version"
-msgstr "Version official"
+msgstr "Versión lanzada"
```

Comment:
¿Qué opinas de "Versión publicada"? No es exactamente lo mismo, pero "Versión lanzada" no es una expresión muy frecuente.

Reply:
"Published version" makes sense to me, and then "unpublished version" is more clear than "Unreleased version".

Reply:
Agree!

Reply:
Much better, like it

Reply:
```

#: development.R:20
msgid "Released version"
msgstr "Versión publicada"

#: development.R:21
msgid "In-development version"
msgstr "Versión en desarrollo"

#: development.R:22
msgid "Unreleased version"
msgstr "Versión no publicada"


```